templates/style/_themes.scss

```diff
@@ -0,0 +1,25 @@
+// Standard white theme
+html {
+  --color-background-code: #f5f5f5;
+  --color-background: #fff;
+  --color-border-light: lighten(#ddd, 5%);
+  --color-border: #ddd;
+  --color-doc-link-background: #333;
```

Comment:
Can you keep the comments from `_vars.scss`? I don't know any of these hex numbers without looking them up.

Reply:
Firefox has a handy little display:

![image](https://user-images.githubusercontent.com/81079/96244155-fb981280-0fa5-11eb-9f36-e21d31eb796b.png)

I think for any style editing it makes sense to prototype it with tools in the browser then copy the values over to the source once it's looking the way you want it to, and use screenshots of the elements in actual usage for review instead of trying to imagine what any changes will look like.

Reply:
Ok, that makes sense.